js_modules/dagster-ui/packages/ui-core/src/assets/__fixtures__/AssetTables.fixtures.ts

```diff
@@ -310,6 +310,7 @@ export const AssetCatalogTableMock = buildQueryMock<
   AssetCatalogTableQueryVariables
 >({
   query: ASSET_CATALOG_TABLE_QUERY,
+  variableMatcher: () => true,
```

Comment:
Is there a way to test this with a low batch size, just to make sure that we're getting the pagination logic correctly, without wildcarding the variables?

Reply:
I'll add a test case for this